server/views/index.ejs

```diff
@@ -0,0 +1,10 @@
+<head>
+	
+</head>
+<body>
+	<div id="app">
+		<h1>Gravity</h1>
+	</div>
+
+	<script src="/static/js/main.bundle.js"></script>
+</body>
```

Comment:
Jeff,
This document is spaced at 4 spaces instead of 2. Please refactor and re-push.